native/cocos/platform/android/java/src/com/cocos/lib/CocosAudio.java

```diff
@@ -32,10 +32,11 @@ of this software and associated engine source code (the "Software"), a limited,
 import android.os.Build;
 import android.util.Log;
 
-class CocosAudioFocusManager {
+public class CocosAudio {
```

Comment:
Why rename this class?

Reply:
it's a module of CocosEngine

Reply:
I think the naming of CocosAudio is too big. Not need to take this renaming.

Reply:
This class is just take care of one thing which is the audio focus logic.